Vagrantfile

```diff
@@ -1,9 +1,20 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+unless Vagrant.has_plugin?("vagrant-service-manager")
+  raise <<-MSG
+    vagrant-service-manager plugin is required for projectatomic/adb.
+    Kindly install the plugin: `$ vagrant plugin install vagrant-service-manager`
+  MSG
+end
+
 Vagrant.configure(2) do |config|
+
   config.vm.box = "projectatomic/adb"
 
   config.vm.network "private_network", type: "dhcp"
 
+  # enable multiple providers as comma separated list. Eg: 'docker, openshift'
+  config.servicemanager.providers = 'docker'
```

Comment:
@navidshaikh 
It is working. :smile: .
Just omit config line from Vagrantfile. [This lin](https://github.com/budhrg/vagrant-service-manager/blob/config-hook/lib/vagrant-service-manager/config.rb#L17) is doing the trick for it.